core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -9328,35 +9328,6 @@ LogicalProject(NAME=[$0], ENAME=[$2])
       Uncollect
         LogicalProject(EMPLOYEES=[$cor1.EMPLOYEES])
           LogicalValues(tuples=[[{ 0 }]])
-]]>
-        </Resource>
-    </TestCase>
-    <TestCase name="testProjectCorrelateTranspose">
-        <Resource name="sql">
-            <![CDATA[select t1.name, t2.ename
-from DEPT_NESTED as t1,
-unnest(t1.employees) as t2]]>
```

Comment:
Note that `testProjectCorrelateTranspose` is redundant[1][2].

[1] https://github.com/apache/calcite/blob/master/core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml#L9226
[2] https://github.com/apache/calcite/blob/master/core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml#L9253